### PR TITLE
Update Chinese Doc duplication and translate it to English

### PR DESCRIPTION
**Pull request by ninsmiracle**

### CODE REVIEW

**Comment by acelyc111** [comment]
Hi @ninsmiracle , many thanks for the contribution!

AFAIK, the Chinese docs are a few outdated, for example:
1. 
```
目前在跨机房网络健康的环境下，数据延时大概在 10s 左右，即 A 机房的写数据大概在 10s 后会写入 B 机房。
```
The 10s can be configured now (by https://github.com/apache/incubator-pegasus/pull/1639), you can add describption like:
```
Since Pegasus 2.6, it can by configured by xxx
```
2. Because FDS is not supported since 2.6, so it would be better to use HDFS as an example to backup data.
3. Because the metric names have been updated since 2.6, it would be better to distinguish the metric names. Or we can do the update in another patch.
4. "完整监控项/HTTP 接口列表" can be removed, because it's not "完整的" and it will cost much time to maintenance the doc when code update.

**Comment by acelyc111** [comment]
It's recommand to update the Chinese docs at first, then translate it to English to reduce duplicate work.

**Comment by ninsmiracle** [comment]
> 目前在跨机房网络健康的环境下，数据延时大概在 10s 左右，即 A 机房的写数据大概在 10s 后会写入 B 机房。

In my opinion,`duplication_sync_period_second` is the second who control the interval time that the duplication info between meta server and replica server.  Intead of control the interval of data dup.
Cause we made some latency tests between master duplication cluster and backup duplication cluster.We write some k-v data to master cluster and each of them less than 1KB,Our results showed that the backup duplication cluster was able to read the data in just about 100ms.

**Comment by acelyc111** [comment]
> > 目前在跨机房网络健康的环境下，数据延时大概在 10s 左右，即 A 机房的写数据大概在 10s 后会写入 B 机房。
> 
> In my opinion,`duplication_sync_period_second` is the parameter who control the interval time that the duplication info between meta server and replica server. Intead of control the interval of data dup. Cause we made some latency tests between master duplication cluster and backup duplication cluster.We write some k-v data to master cluster and each of them less than 1KB,Our results showed that the backup duplication cluster was able to read the data in just about 100ms.

Thanks for the information! Is it the fixed time interval in https://github.com/apache/incubator-pegasus/blob/master/src/replica/duplication/duplication_pipeline.cpp#L63 ?

The docs is needed to be updated anyway.

File: _docs/zh/administration/duplication.md [hunk 1]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
...
+  [[duplication-group]]
+  bjsrv-account = 1
+  tjsrv-account = 2
 ```
 
-### 执行步骤4
 
-现在我们启动热备份。
+
+### 执行步骤2 按需接入域名proxy系统
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:150]: 需要明显说明这个是可选的
```suggestion
### 执行步骤2 按需接入域名proxy系统（可选）
```

File: _docs/zh/administration/duplication.md [hunk 2]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
...
+
+跨机房热备的主要目的是提供机房级容灾，为了提供跨机房切换流量的能力，在内部使用中需要热备的业务必须接入meta-proxy。
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:152]: 官网中没有任何地方说明 meta-proxy 是什么，也没有加在Pegasus主项目中。你可以留个TODO后面来做

File: _docs/zh/administration/duplication.md [hunk 3]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
...
-    id         : 1541649698875
-    start_time : 2018-11-08 12:01:38
-    end_time   : 2018-11-08 12:03:51
-    app_ids    : {60}
-
-# 在天津机房恢复表
-./run.sh shell -n tjsrv-account
->>> restore_app -c bjsrv-account -p dup_transfer -a account_xiaomi -i 12 -t 1541649698875 -b fds_wq
+
+
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:130]: 留两个空行就行了？

File: _docs/zh/administration/duplication.md [hunk 4]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
...
+successfully add duplication [dupid: 1669972761]
 ```
 
-当 `start_dup` 时，热备份任务会从之前记录的确认点开始复制，这样我们就保证了写增量的完整性。
 
-另外需注意的是，由于写增量的长时间堆积，一时可能有大量日志复制，热备份流量会突增，从而导致服务不稳定。因此，我们需要在远端机房设置[限流（write throttling）](/administration/throttling)。
 
+### 执行步骤4 暂停/重启/删除一个热备任务
+
+```C#
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:181]: 为啥是 C#

File: _docs/zh/administration/duplication.md [hunk 5]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
...
+
+当然如果业务侧可以自己修改meta地址，是可以不接入域名proxy系统的。
+
+
+
+### 执行步骤3 开启热备
+
+在开启热备前，需要考虑好本次热备是同步表的全部数据(全量数据同步)还是只需要同步此刻开始(增量同步)。
+
+1. 如果进行全量拷贝，则需要拷贝checkpoint，则备集群不能存在同名表，由该命令创建新表并开启主备任务
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:164]: "拷贝checkpoint"  不是通过HDFS备份回复来做吗？  "拷贝checkpoint"这个操作对于新人没有任何文档参考

还是需要明确说明怎么操作的

File: _docs/zh/administration/duplication.md [hunk 6]
```diff
@@ -242,15 +254,18 @@ set_dup_fail_mode <app_name> <dupid> <slow|skip>
...
 [duplication-group]
-  tjsrv-account = 1
-  bjsrv-account = 2
+  bjsrv-account = 1
+  tjsrv-account = 2
+  
 ```
 
-我们在每条数据前都会加上 `timestamp+cluster_id` 的前缀，timestamp 即数据写到 pegasus 的时间戳，cluster_id 即上面 duplication-group 中所配置的，tjsrv 的 cluster_id 为 1，bjsrv 的 cluster_id 为 2。
+我们在每条数据前都会加上 `timestamp+cluster_id` 的前缀，timestamp 即数据写到 pegasus 的时间戳，cluster_id 即上面 duplication-group 中所配置的，bjsrv 的 cluster_id 为 1，tjsrv 的 cluster_id 为 2。
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:262]: bjsrv 和 tjsrv 跟上面的集群完整名字保持一致吧

File: _docs/zh/administration/duplication.md [hunk 7]
```diff
@@ -242,15 +254,18 @@ set_dup_fail_mode <app_name> <dupid> <slow|skip>
...
 
 cluster_id 的作用是：一旦出现写冲突，例如 tjsrv 和 bjsrv 同时写 key `"user_1"`，系统首先会检查两次写的时间戳，以时间戳大的为最终值。当极罕见地遇到时间戳相同的情况时，以 cluster_id 大的为最终值。使用这种机制我们可以保证两集群的最终值一定相同。
 
-## 完整监控项列表
+
+
+## 监控项列表
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:268]: 因为名字全都改了，可以该成说 “带dup关键字的指标项”，或直接删除整段（新metrics框架会有每个指标项的说明，此处去说反而带来困扰，难以维护）

File: _docs/zh/administration/duplication.md [hunk 8]
```diff
@@ -33,33 +33,43 @@ Type "help" for more information.
...
+    "create_ts": 1692240106066,
+    "fail_mode": "FAIL_SLOW"
+  }
+]
 ```
 
-通过 `add_dup` 命令，bjsrv-account 集群的表 account_xiaomi 将会近实时地把数据复制到 tjsrv-account 上，这意味着，每一条在北京机房的写入，最终都一定会复制到天津机房。
+通过 `dup add` 命令，bjsrv-account 集群的表 my_source_app 将会近实时地把数据复制到 tjsrv-account 上，这意味着，每一条在北京机房的写入，最终都一定会复制到天津机房。
 
 热备份使用日志异步复制的方式来实现跨集群的同步，可与 mysql 的 binlog 复制和 hbase replication 类比。
```
Comment (acelyc111) [inline _docs/zh/administration/duplication.md:63]:
```suggestion
热备份使用日志异步复制的方式来实现跨集群的同步，可与 MySQL 的 binlog 复制和 HBase replication 类比。
```

File: _docs/zh/administration/duplication.md [hunk 9]
```diff
@@ -72,110 +82,110 @@ duplications of app [account_xiaomi] are listed as below:
 面对这个需求，我们的操作思路是：
 
 1. 首先源集群**保留从此刻开始的所有写增量**（即WAL日志）
-2. 将源集群的全量快照（冷备份）上传至 HDFS / xiaomi-FDS 等备份存储上。
-3. 然后恢复到目标集群。
-4. 此后源集群开启热备份，并复制此前堆积的写增量，复制到远端目标集群。
+2. 将源集群的全量快照（存量数据）移动到指定路径下，等待备集群(目标集群)对这些数据进行学习learn。
```
[the remark this answers is not in the document]
Comment (acelyc111) [reply _docs/zh/administration/duplication.md:85]: 对于冷备份，改用“备份/backup”“恢复/restore”的称谓吧？

**Comment by acelyc111** [comment]
https://github.com/apache/incubator-pegasus/issues/1745

File: _docs/zh/administration/duplication.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (ninsmiracle) [reply _docs/zh/administration/duplication.md:85]: > 存量数据还是需要用HDFS冷备份的方式吧

最新版本的热备不需要了，主要是通过learn来进行dup的。所以我将全量checkpoint携程了存量数据

File: _docs/zh/administration/duplication.md (hunk 3 above)
Comment (ninsmiracle) [reply _docs/zh/administration/duplication.md:130]: 是的，因为现在存量数据不走外部hdfs或者fds了。 第一步只需要在集群配置中加一些项

**Comment by ninsmiracle** [comment]
THX for the comment and sorrry about the untimely reply. Cause we have too many online failure recently...

**Review by acelyc111 — APPROVED** [review]

**Pull request merged**